thoth/slo_reporter/sli_report.py

```diff
@@ -33,6 +34,7 @@
 _START_TIME_EPOCH = int(_START_TIME.timestamp() * 1000)
 _END_TIME_EPOCH = int(_END_TIME.timestamp() * 1000)
 
+_ENVIRONMENT = os.environ["THOTH_ENVIRONMENT"]
```

Comment:
is this set in the cronjob?

Reply:
yes
```
  - description: Environment in which Thoth is running
    displayName: Thoth Enviroment
    required: true
    name: THOTH_ENVIRONMENT

                    - name: THOTH_ENVIRONMENT
                      value: "${THOTH_ENVIRONMENT}"
```
 it should come from CM `slo-reporter`